src/modules/views/Folder/FolderViewBody.jsx

```diff
@@ -115,6 +132,11 @@ const FolderViewBody = ({
 
   const { syncingFakeFile } = useSyncingFakeFile({ isEmpty, queryResults })
 
+  const onToggleSelect = (fileId, e) => {
+    setLastInteractedItem(fileId)
+    onShiftClick(fileId, e)
```

Comment:
the goal here is to trigger shift selection. Is it possible to use only on func `onShiftClick`? Is `setLastInteractedItem` useful for something else?

Reply:
ok. Maybe the main problem here is that is hard to understand what the API is and how use it. But jsdocs helps a lot :)